spec/rubocop/node_pattern_spec.rb

```diff
@@ -1173,6 +1189,12 @@ def withargs(foo, bar, qux)
       it_behaves_like 'invalid'
     end
 
+    context 'with unmatched opening paren and ...' do
```

Comment:
It becomes more clearer that it is code when `...` is enclosed in backticks

```diff
-context 'with unmatched opening paren and ...' do
+context 'with unmatched opening paren and `...`' do
```

Reply:
Fixed.